src/py/covidr/database/schema.py

```diff
@@ -0,0 +1,28 @@
+import sqlalchemy.event as sqlalchemy_event
+from sqlalchemy import DDL
+
+from .connection import session_scope, SqlAlchemyInterface
+from .models import meta
+
+
+def create_tables_and_schema(interface: SqlAlchemyInterface):
+    """Creates the schema and all tables"""
+    with session_scope(interface) as session:
+        sqlalchemy_event.listen(
+            meta,
+            "before_create",
+            DDL(f"CREATE SCHEMA IF NOT EXISTS {meta.schema}"),
+        )
```

Comment:
oh ok interesting, why not just create the schema in the initial migration?

Reply:
The reason is that I want `make init-local-db` actually produce a functional db.